src/index.js

```diff
@@ -3,9 +3,63 @@
 
 // An example of how you tell webpack to use a CSS (SCSS) file
 import './css/base.scss';
-import datepicker from 'js-datepicker'
+import Hotel from './Hotel';
+
 
 // An example of how you tell webpack to use an image (also need to link to it in the index.html)
 import './images/alexander-kaunas-67-sOi7mVIk-unsplash.jpg'
+import User from './User';
+
+
+const totalStays = document.getElementById('numberStays');
+const totalSpent = document.getElementById('totalSpent');
+const userHistory = document.getElementById('userHistory');
+const userName = document.getElementById('userName'); 
+let currentUser;
+let hotel;  
+
+const apiData = [fetch('http://localhost:3001/api/v1/customers/1'), fetch('http://localhost:3001/api/v1/rooms'), fetch('http://localhost:3001/api/v1/bookings')]
+
+Promise.all(apiData)
```

Comment:
I'm not sure what the best practice is for when to all a GET request, here it seems like it's being called on page load? Maybe move this to a function? Not sure though..